2022/36xxx/CVE-2022-36085.json

```diff
@@ -1,18 +1,116 @@
 {
```

Comment:
'https://github.com/open-policy-agent/opa/pull/4616 ' does not match '^(ftp|http)s?://\\S+$'